docs/polaris/applications-and-libraries/applications/vasp.md

```diff
@@ -110,11 +110,15 @@ BINDIR     = ../../bin
 The follow modules will update the include and libraries paths used by the Cray compiler wrapper `ftn` to load additional math libraries for the CPU.
 
 ```
-module purge
-module load nvhpc/23.3
+module restore
 module load PrgEnv-nvhpc
 module load cray-libsci
 module load craype-accel-nvidia80
+export NVROOT=/opt/nvidia/hpc_sdk/Linux_x86_64/23.9
```

Comment:
Using $NVIDIA_PATH provided by module would be cleaner.

knight@polaris-login-04:~> echo $NVIDIA_PATH
/opt/nvidia/hpc_sdk/Linux_x86_64/23.9